src/DependencyManagement/DependencyManager.cs

```diff
@@ -60,6 +74,74 @@ internal DependencyManager()
             Dependencies = new List<DependencyInfo>();
         }
 
+        /// <summary>
+        /// Processes the dependency download request
+        /// </summary>
+        /// <param name="msgStream">The protobuf messaging stream</param>
+        /// <param name="request">The StreamingMessage request for function load</param>
+        internal void ProcessDependencyDownload(MessagingStream msgStream, StreamingMessage request)
+        {
+            if (request.FunctionLoadRequest.ManagedDependencyEnabled)
+            {
+                var rpcLogger = new RpcLogger(msgStream);
+                rpcLogger.SetContext(request.RequestId, null);
+                if (Dependencies.Count == 0)
+                {
+                    // If there are no dependencies to install, log and return.
+                    rpcLogger.Log(LogLevel.Trace, PowerShellWorkerStrings.FunctionAppDoesNotHaveDependentModulesToInstall, isUserLog: true);
```

Comment:
Could it be possible that the portal looks like freezing currently without any message only because we are using the `Trace` level logging? Maybe if you change the current logging to `Information` level and the message will show up even without this change?

Reply:
I tried changing the trace level but it doesn't show up on azure portal unless it's exception.